scripts/globals/events/harvest_festivals.lua

```diff
@@ -2,16 +2,20 @@
 -- Harvest Festivals
 -----------------------------------
 require("scripts/globals/utils")
+require("scripts/globals/npc_util")
 -----------------------------------
+xi = xi or {}
+xi.events = xi.events or {}
+xi.events.harvest = xi.events.harvest or {}
 
-function isHalloweenEnabled()
+xi.events.harvest.isHalloweenEnabled = function()
     local option = 0
     local month = tonumber(os.date("%m"))
     local day = tonumber(os.date("%d"))
 
     if
-        month == 10 and day >= 20 or
-        month == 11 and day == 1 or
+        month == 1 and day >= 20 or
+        month == 12 and day == 1 or
```

Comment:
We're making this run December + January? Wouldn't that overlap with starlight celebration festival?